docker/salt/tethyscore.sls

```diff
@@ -5,67 +5,125 @@
 {% set CLIENT_MAX_BODY_SIZE = salt['environ.get']('CLIENT_MAX_BODY_SIZE') %}
 {% set ASGI_PROCESSES = salt['environ.get']('ASGI_PROCESSES') %}
 {% set TETHYS_BIN_DIR = [CONDA_HOME, "/envs/", CONDA_ENV_NAME, "/bin"]|join %}
+{% set TETHYS_DB_NAME = salt['environ.get']('TETHYS_DB_NAME') %}
 {% set TETHYS_DB_HOST = salt['environ.get']('TETHYS_DB_HOST') %}
 {% set TETHYS_DB_PASSWORD = salt['environ.get']('TETHYS_DB_PASSWORD') %}
+{% set POSTGRES_PASSWORD = salt['environ.get']('POSTGRES_PASSWORD') %}
 {% set TETHYS_DB_PORT = salt['environ.get']('TETHYS_DB_PORT') %}
 {% set TETHYS_DB_USERNAME = salt['environ.get']('TETHYS_DB_USERNAME') %}
 {% set TETHYS_HOME = salt['environ.get']('TETHYS_HOME') %}
+{% set TETHYS_PORT = salt['environ.get']('TETHYS_PORT') %}
 {% set TETHYS_PUBLIC_HOST = salt['environ.get']('TETHYS_PUBLIC_HOST') %}
 {% set TETHYS_SUPER_USER = salt['environ.get']('TETHYS_SUPER_USER') %}
-{% set TETHYS_SUPER_USER_EMAIL = salt['environ.get']('TETHYS_SUPER_USER_EMAIL') %}
```

Comment:
Why did we get rid of the TETHYS_SUPER_USER_EMAIL?

Reply:
Similarly the TETHYS_SUPER_USER and TETHYS_SUPER_USER_PASS have also been replaced with PORTAL_SUPERUSER_ counterparts. Can we make this consistent throughout?